Makefile

```diff
@@ -162,6 +166,27 @@ verify:
 	echo "Checking commit integrity"
 	$(MAKE) verify-commits
 
+.PHONY: crc-start
+crc-start:
+	echo "Starting CRC"
+	./scripts/crc-start.sh
+
+.PHONY: crc-build
+crc-build:
+	echo "Building olm image"
+	IMG="olm:test" $(MAKE) build/olm-container
+	echo "Building opm image"
+	rm -rf bin
```

Comment:
Any idea on why this rm command was needed between build steps?

Reply:
I'll add a comment. But basically, if we don't rm bin before building the opm container and the binaries are still in bin, it won't (re-)compile them and you'll end up with a binary that doesn't work inside the container =S

Reply:
I'm cleaning this up by adding a clean target and calling it before the build/*-container targets